warp10/src/main/java/io/warp10/script/functions/TIMEBOX.java

```diff
@@ -45,31 +45,46 @@
    * Maximum timeboxing possible
    */
   private static final long TIMEBOX_MAXTIME;
-  
   static {
     TIMEBOX_MAXTIME = Long.parseLong(WarpConfig.getProperty(Configuration.CONFIG_WARPSCRIPT_TIMEBOX_MAXTIME, Long.toString(DEFAULT_TIMEBOX_MAXTIME)));
   }
-  
+
+  /**
+   * Allowance capability to raise TIMEBOX_MAXTIME
+   */
+  private static final String SPECIAL_ALLOWANCE_CAPNAME = Configuration.CONFIG_WARPSCRIPT_TIMEBOX_MAXTIME;
```

Comment:
Use a specific configuration key which will contain the actual capability name to use for extending timebox maxtime. If such key is not set then no capability will be checked.